packages/thumbprint-react/components/Image/components/picture.jsx

```diff
@@ -1,14 +1,29 @@
-import React, { forwardRef } from 'react';
+import React, { forwardRef, useState } from 'react';
+import styles from './picture.module.scss';
 
 const Picture = forwardRef((props, ref) => {
     const { src, sources, sizes, alt, style, ...rest } = props;
 
+    const [loadedClass, setClass] = useState(null);
```

Comment:
This is a bit easier to read IMO if it's:

```js
const [isLoaded, setIsLoaded] = useState(false);
```

And then in the `img`:

```jsx
<img
    src={src}
    alt={alt}
    style={style}
    ref={ref}
    {...rest}
    onLoad={() => { setIsLoaded(true) }}
    className={isLoaded ? styles.imageLoaded : styles.imageLoading}
/>
```

I think it works better since `isLoaded` more accurately describes the "state" of the image whereas `loadedClass` is very specific to what we intend to do with that state.

Reply:
Yeah, that's cleaner. Just updated.